src/JobModal/JobModal.js

```diff
@@ -0,0 +1,90 @@
+import {useEffect, useState} from "react";
+import ContractTypeTag from "../ContractType/contractType";
+import SkillTag from "../SkillTag/SkillTag";
+import uniqid from "uniqid";
+
+const JobModal = ({jobId, modalShow, setModalShow}) => {
+    const [jobData, setJobData] = useState(null)
+
+    const closeModal = () => {
+        setModalShow(false)
+    }
+
+    const modalClick = (e) => {
+        e.stopPropagation()
+    }
+
+    useEffect(() => {
+        const getSingleJob = async () => {
+            let response = await fetch(`http://localhost:8080/jobs/${jobId}`)
+            const jobData = await response.json()
+            setJobData(jobData)
+        }
+        if(jobId !== null){
+            getSingleJob()
+        }
+
+    }, [jobId])
+
+    function formatDate(dateStr) {
+        let dateArray = dateStr.split("-")
+        let newDate = dateArray[2] + '/' + dateArray[1] + '/' + dateArray[0]
+        return newDate
+    }
+
+    return (
+        <>
+            {jobData &&
+                <div onClick={closeModal} className={modalShow === true ? 'modalScreen modalOpen' : 'modalScreen modalClosed'}>
+                    <section onClick={modalClick} className="JobModal">
+                        <div className="modalTitle">
+                            <h3>{jobData.job_title} - {jobData.company}</h3>
+                            <a href="#"><p onClick={closeModal}>{"\u2715"}</p></a>
```

Comment:
The href attribute requires a valid value to be accessible. Provide a valid, navigable address as the href value. If you cannot provide a valid href, but still need the element to resemble a link, use a button and change it with appropriate styles. Learn more: https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/HEAD/docs/rules/anchor-is-valid.md